packages/server/lib/refreshConnections.ts

```diff
@@ -56,21 +56,22 @@ export async function exec(): Promise<void> {
             const limit = 1000;
             // eslint-disable-next-line no-constant-condition
             while (true) {
-                const staleConnections = await connectionService.getStaleConnections({ days: 0, limit, cursor });
+                const staleConnections = await connectionService.getStaleConnections({ days: 1, limit, cursor });
```

Comment:
I think it was an error, let me know @TBonnin

Reply:
yes, that's a test leftover. Thank you for fixing